bot/src/main/java/edu/java/bot/commands/ListCommand.java

```diff
@@ -0,0 +1,43 @@
+package edu.java.bot.commands;
+
+import com.pengrad.telegrambot.TelegramBot;
+import com.pengrad.telegrambot.model.Message;
+import edu.java.bot.models.Users;
+import java.util.List;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+@Component("/list")
```

Comment:
лучше имя не писать в имя бина, лучше делать через метод. 
Проблема в том, что в случае если, у тебя будут два бина с одинаковым именем, то спринг тупо одним перезатрёт другой и ничего не скажет, поэтому убирай
```suggestion
@Component
```